src/main/java/org/haffson/adventofcode/days/Days.java

```diff
@@ -17,7 +18,7 @@ public interface Days {
      *
      * @return the solution of Part 1
      */
-    String firstPart();
+    String firstPart() throws FileNotFoundException;
```

Comment:
If you do not want to (or cannot) modify an interface in such a case, then you can wrap a checked exception into any kind of `RuntimeException`.